### ISSUE TITLE
Government design

### ISSUE THREAD

**Issue by GNY-001F2** [TheCityModTeam/designdocumentation#1]
So we're representing governments, businesses and criminals.
- Government (unplayable)
  - Municipal corporation
    - Mayor
    - Ward councillors (college of cardinals)
    - We can recycle catholic mechanics for this I think
    - Party office baronies can replace bishoprics
  - Police
    - Succession like mercenary companies/holy orders
  - Court of justice
    - punishment for crimes, and settlement of some kinds of disputes that will replace wars
- Businesses (design WIP)
  - Enterprise
    - locked to primogeniture
    - easy to get funding early on but becomes worse if you don't upgrade
    - titular, locked to count-level
    - protected from attacks by gangs, but limited ability to expand after initial capital runs out
  - Family business
    - open, gavelkind, primo/ultimo and tanistry
    - vassals are only dynasty members
    - easier to branch out, easier to fragment, but also gives good alliances if we can enable 2.4.5-style alliances
    - up to king level
  - Private company
    - elective; if possible I'd like AI to choose director with the most titles
    - duke or king level title
  - Public corporation
    - king level title; special mechanics to unlock emperor level title and deactivate the mayor
    - patrician elective
    - vassals are subsidiary companies
  - Subsidiary
    - special vassal company - characters get greater autonomy but lieges can revoke these titles
  - Mergers and acquisitions replace vassalization
  - honorary titles are replaced by vice presidents
  - hired employees are represented by council (CEO, CFO, secretary, attorney etc)
    - cannot revoke your vassals except in public corps as they are directors in the company - but you can spin them off.
  - Law firm (unplayable)
    - replaces holy orders?
  - Security company (unplayable)
    - replaces mercenary bands
- Criminals (WIP)
  - Unorganized crime (tribal)
    - Gangs
      - Modified tribal mechanics at the start, but very weak upon succession.
      - Locked to the rank of count
  - Organized crime (feudal)
    - Crime family
    - Cartel
    - ???

What other kinds of governments do you guys think should be in here?

It doesn't have to be identical to how things work in real life cities. Some amount of leeway for 👻 and &#9651; should be fun.

What are your thoughts?

**Comment by Shoall** [comment]
For OC(organised crime) we can also add smuggling rings and other bussineses wich are used as a cover up for other crimes

**Comment by Darokrithia** [comment]
I feel like government type people, especially the mayor (say, if you fund their campaign), should be able to be added to a tributary like diplomatic status in which they grant benefits, such as lowered taxes, lowered regulations, ability to remove criminal charges (which possibly should be negative traits), and the ability to get people out of prison.

Organized crime also should be able to be the surizens of small businesses, which makes them defend the smaller businesses, but forces them to pay taxes.

Thats all I can think of right now.

**Comment by GNY-001F2** [comment]
That sounds like protection rackets @Darokrithia. I like it!

I think criminal charges and stuff should be handled by the police chief and the mayor used as a layer of indirection to get the right police chief appointed to the job. Also the court of justice as another resort.

Mostly for the criminal types we need to figure out compelling features for each type otherwise some of the governments would not be preferable to others if the gameplay other than succession laws are the same. It will end up becoming like in CK2 where we usually get rid of gavelkind ASAP when the game starts in favour of other forms.

I think criminal accusations should be traits, and convictions should result in actual penalties, imprisonment being one, cash penalty another etc.

**Comment by Darokrithia** [comment]
I agree with the trait system, that all sounds good.

I also was playing some After the End last weekend.  While we should use pope mechanics for the mayor to start out, After the End has a way of letting / getting every adult americanist to vote on presidential candidates.  We should message the AtE modders and see if we can view or borrow their code.

**Comment by GNY-001F2** [comment]
That sounds like a good idea.

My idea was that only the councillors can vote the mayor in since the mayor enjoys his office when he has the confidence of the city council - this is common in many countries with a parliamentary system of government, especially in the Commonwealth of Nations.

However - if it is possible to do so I think the councillors should be elected by everyone who falls within that ward/zone's jurisdiction. Here maybe the presidential election mechanics in AtE could be a useful reference. I'll drop them a message for permission, and see what they say.

**Comment by GNY-001F2** [comment]
Also I'm against directly copying/borrowing their code. I was interested in releasing the work we do here under the GPL or MIT license so that people don't have to ask us for permission to use our work. Their code falls under standard copyright and I'd rather avoid the legal headache.

**Comment by GNY-001F2** [comment]
Just thought up conglomerates:

They will be like public corporations but even bigger and more powerful. We can use nomadic government mechanics here.

**Comment by Darokrithia** [comment]
I agree with that.  Don't copy, just ask for snippets (Im sure they will oblige).

**Comment by Darokrithia** [comment]
I like the nomad ideas, we could also do merchant republic.  Each of the "Khans" should be major stock holders.  When companies fuse (through a subjugation type war or through diplomacy), the khans from both groups should join into one "nomadic" empire.

**Comment by GNY-001F2** [comment]
They already gave us permission to view the code. After the end is on github and I created a thread there. They also said they're okay with us copying their work, but I'll ask them to clarify about licensing.

**Comment by GNY-001F2** [comment]
I think Yakuza-style government should be represented with nomadic clans, as different branches.

**Comment by Darokrithia** [comment]
I think it might be cool to have a trait that is passed down from your father that is your "clan" if we are doing clan mechanics.

**Comment by GNY-001F2** [comment]
Why would clan have to be passed down as traits? Nomadic governments already take care of that.

**Comment by Darokrithia** [comment]
It would let clan members outside of the court of clan leaders to continue being loyal (or try to seize power from the clan).  It was just a secondary idea.  It also could be used for people who do not own HL

**Comment by GNY-001F2** [comment]
Hmm, yeah let's aim for compatibility with the core game minus DLC first. I haven't actually played with the DLC disabled before, so I'll try to get some playtime in with just the core game next week to figure out how much we can't do without.

**Comment by Darokrithia** [comment]
Im really sorry guys, but I wont be able to work on this much on this until late june.  Also, I think that we should use a horse lord like system of little to no DLC needed, but with DLCs being used if you have them AGOT does this very well.

**Comment by GNY-001F2** [comment]
@Darokrithia yeah, I agree. Also no worries, I have stuff too this month so I'm taking it a bit slow. Try to chip in as much as you can.

For now let us first finalize the design and then start writing more substantial code.
